tools/influx_si/influx_si.xml

```diff
@@ -1,232 +1,74 @@
-            $opt.TIMEIT
-            #if $opt.tblimit:
-             --tblimit='$opt.tblimit'
+<tool id="influx_si" name="influx_si" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.09">
+    <description>
+        Estimate metabolic fluxes and concentrations by fitting simulated labeling in metabolites to NMR/MS measurements
+    </description>
+    <creator>
+        <person givenName="Serguei" familyName="Sokol" email="sokol@insa-toulouse.fr" url="https://www.toulouse-biotechnology-institute.fr/en/plateformes-plateaux/cellule-mathematiques/" />
+        <organization name="INRAE" url="https://www.inrae.fr/" />
+        <organization name="TBI" url="https://www.toulouse-biotechnology-institute.fr/" />
+        <organization name="Mathematics Cell" url="https://www.toulouse-biotechnology-institute.fr/en/plateformes-plateaux/cellule-mathematiques/" />
+        <organization name="MetaToul-FluxoMet" url="https://www.toulouse-biotechnology-institute.fr/en/plateformes-plateaux/metatoul/" />
+        <organization name="MetaboHub2" url="https://www.metabohub.fr/" />
+    </creator>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">influx_si</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir outdir;
+        #for $inp in $input_main:
+            unzip $inp;
+        #end for
+               
+        ret=0;
+        influx_$si.s_i
+            @COMMAND_OPT@
+            #if $si.s_i == 'i' and $si.time_order != 'None':
+              --time_order='$si.time_order'
             #end if
+            #for $inp in $input_main:
+                #set base = $inp.name[:-4]
+                --prefix "$base"
+            #end for
+            @MTF_OPT@
+            > 'outdir/influx_${si.s_i}.stdout.txt' 2>'outdir/influx_${si.s_i}.stderr.txt' ||
+            ret=\${PIPESTATUS[0]};
+
 
-         #if $si.s_i == 'i' and $si.time_order:
-           --time_order='$si.time_order'
-         #end if
-         '${"' '".join([str($in_m.element_identifier) for $in_m in $input_main])}'
-         > 'influx_${si.s_i}.stdout.txt' 2>'influx_${si.s_i}.stderr.txt'; res="\$?" &&
-       
-       for f in *.ftbl *.log *.err *.R *.kvh *.attrs; do mv -f "\$f" "\$f".txt || true; done &&
-       
-       for f in *.pdf; do [ -e "\$f" ] && mv -f "\$f" "\$f".pdf || true; done &&
-       
-       for f in *.RData; do [ -e "\$f" ] && mv -f "\$f" "\$f".RData || true; done &&
-       [ "\$res" == "0" ]
+        #for $inp in $input_main:
+            #set base = $inp.name[:-4]
+            for f in "$base"_res/*.{err,log}; do mv -f "\$f" "\$f".txt || true; done;
+            for f in "$base"_res/*.{sim,stat}; do mv -f "\$f" "\$f".tsv || true; done;
+            for f in "$base"_res/*.pdf; do [ -e "\$f" ] && mv -f "\$f" "\$f".pdf || true; done;
+            rm -rf "$base"_res/tmp/;
+            mv -f "$base"_res/* outdir/;
```

Comment:
Maybe you can also simplify this a bit

```suggestion
            mv -f "$base"_res/*.* outdir/;
```

that is, just copy the files and leave the `tmp` dir where it is.

Reply:
In fact, `tmp` dir must be removed as it can take a lot of disk-space and clutter a beginner user's vision with files difficult to interpret.

Reply:
`tmp` won't be discovered by Galaxy except if you set `recurse="true"`  in `<discover_datasets>`. So users do not see it anyway.